ql/src/java/org/apache/hadoop/hive/ql/ddl/process/show/compactions/ShowCompactionsOperation.java

```diff
@@ -77,7 +78,7 @@ private void writeHeader(DataOutputStream os) throws IOException {
     os.write(Utilities.tabCode);
     os.writeBytes("State");
     os.write(Utilities.tabCode);
-    os.writeBytes("Hostname");
+    os.writeBytes("Worker hostname");
```

Comment:
maybe use just `Worker host`, same for Initiator

Reply:
fixed